src/project.js

```diff
@@ -121,6 +123,15 @@ function project(location = {}, targetAltitudinalZone) {
     );
   }
 
+  newLocation.options = {
+    ...newLocation.options,
+    forestEcoregion: Array.from(new Set(newLocation.options.forestEcoregion)),
```

Comment:
Let's do this directly when doing the "concat", because we can live with a minor performance impact. We should try to avoid specifying each field again, because the for-loop already iterates over all of them.

Reply:
Maybe we should also sort the options?

Reply:
Cool idea!